tests/integration/post_training/test_post_training.py

```diff
@@ -39,62 +77,74 @@ async def test_supervised_fine_tune(self, post_training_stack):
         )
 
         data_config = DataConfig(
-            dataset_id="alpaca",
+            dataset_id=dataset.identifier,
             batch_size=1,
             shuffle=False,
+            data_format="instruct",
         )
 
-        optimizer_config = OptimizerConfig(
-            optimizer_type="adamw",
-            lr=3e-4,
-            lr_min=3e-5,
-            weight_decay=0.1,
-            num_warmup_steps=100,
-        )
-
+        # setup training config with minimal settings
         training_config = TrainingConfig(
             n_epochs=1,
             data_config=data_config,
-            optimizer_config=optimizer_config,
             max_steps_per_epoch=1,
             gradient_accumulation_steps=1,
         )
-        post_training_impl = post_training_stack
-        response = await post_training_impl.supervised_fine_tune(
-            job_uuid="1234",
-            model="Llama3.2-3B-Instruct",
+
+        job_uuid = f"test-job{uuid.uuid4()}"
+        logger.info(f"Starting training job with UUID: {job_uuid}")
+
+        # train with HF trl SFTTrainer as the default
+        _ = llama_stack_client.post_training.supervised_fine_tune(
```

Comment:
is this test going to run on all PRs?

Reply:
let me see if a smaller model runs the test any faster

Reply:
@ashwinb figured it out! the ubuntu runners have intel CPUs and I had to add an intel specific optimization to the SFTConfig, should run in under 2 mins now.